pymorphy3/analyzer.py

```diff
@@ -68,7 +73,7 @@ def __init__(self, dict_path):
         cpd_path = os.path.join(dict_path, 'p_t_given_w.intdawg')
         self.p_t_given_w = ConditionalProbDistDAWG().load(cpd_path)
 
-    def apply_to_parses(self, word, word_lower, parses):
+    def apply_to_parses(self, word: str, word_lower: str, parses: List[Parse]) -> Union[List[Parse], List[_Parse]]:
         if not parses:
             return parses
```

Comment:
It never returns a nonempty list of `Parse`. Just `return []` here and change return type to `-> List[_Parse]`.